share/completions/lxc.fish

```diff
@@ -0,0 +1,51 @@
+function __fish_lxc_no_subcommand -d 'Test if lxc has yet to be given the command'
+    for i in (commandline --tokenize --cut-at-cursor --current-process)
+        if contains -- $i config copy delete exec file help image launch list move network profile publish remote restore restart snapshot start stop
+            return 1
+        end
+    end
+    return 0
+end
+
+function __fish_lxc_list_containers
+    lxc list -c n | grep '| \w' | sed -e 's/\(|\| \)*//g'
+end
+
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments config --description 'Manage configuration.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments copy --description 'Copy containers within or in between lxd instances.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments delete --description 'Delete containers or container snapshots.'
```

Comment:
I agree with you that that text would be better. However, I took these texts directly from the `lxc help <command>` output. Leaving it would mean the original output is the One And Only Truth. I kinda like that too. 
I'm a little on the fence here.